### PR TITLE
add the cart controller and update the cart router

### PR DESCRIPTION
**Pull request by sudharaD**
Fixed the issue #19

### CODE REVIEW

File: controllers/cart.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+//Get all the carts
+exports.getCarts = async(req,res)=>{
+
+    try {
+        // TODO : database call
+        let carts;
+        
+        if(!carts) {
+            res.status(404).send({msg:'Sorry, there are no sections !',  data:[]})
```
Comment (hello-work) [inline controllers/cart.js]:
```suggestion
            res.status(404).send({msg:'Sorry, there are no carts !',  data:null})
```

File: controllers/cart.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    // destructure the data according to carts schema
+    const { data } = req.body;
+
+    try {
+        // TODO : database call
+        let cart;
+
+        if(!cart) {
+            res.status(400).send({msg:'Sections not created', data: []})
```
Comment (hello-work) [inline controllers/cart.js]:
```suggestion
            res.status(400).send({msg:'carts not created', data: null})    
```

File: controllers/cart.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+// get cart
+exports.getCart = async(req,res)=>{
+    const { id } = req.params;
+   
+    try {
+        // TODO : database call
+        let cart;
+
+        if(!cart) {
+            res.status(404).send({msg:'Section Not Found',  data:[]})
```
Comment (hello-work) [inline controllers/cart.js]:
```suggestion
            res.status(404).send({msg:'cart Not Found',  data:null})
```

File: controllers/cart.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+// update cart
+exports.updateCart = async(req,res)=>{
+    const {data} = req.body
+    const {id} = req.params
+
+    try {
+        // TODO : database call
+        let cart;
+
```
Comment (hello-work) [inline controllers/cart.js]:
```suggestion
if(!cart){
   res.status(404).send({msg:'cart Not Found',  data:null})
   }
```

File: controllers/cart.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+//Delete Section
+exports.deleteCart = async(req,res)=>{
+    const {id} = req.params
+
+    try {
+        // TODO : database call
+        let cart;
+
```
Comment (hello-work) [inline controllers/cart.js]:
```suggestion
if(!cart){
   res.status(404).send({msg:'cart Not Found',  data:null})
 }
```

File: controllers/cart.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+exports.updateCart = async(req,res)=>{
+    const {data} = req.body
+    const {id} = req.params
+
+    try {
+        // TODO : database call
+        let cart;
+if(!cart){
+   res.status(404).send({msg:'cart Not Found',  data:null})
+   }
```
Comment (sudharaD) [inline controllers/cart.js]:
```suggestion
      if(!cart){
         res.status(404).send({msg:'cart Not Found',  data:null})
      }
```

File: controllers/cart.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+//Delete Section
+exports.deleteCart = async(req,res)=>{
+    const {id} = req.params
+
+    try {
+        // TODO : database call
+        let cart;
+if(!cart){
+   res.status(404).send({msg:'cart Not Found',  data:null})
+ }
```
Comment (sudharaD) [inline controllers/cart.js]:
```suggestion
          if(!cart){
             res.status(404).send({msg:'cart Not Found',  data:null})
           }
```

**Review by hello-work — COMMENTED** [review]
Approved and merged

**Pull request merged**